src/common/http.cc

```diff
@@ -455,9 +463,15 @@ namespace Pistache::Http
                     if (!cursor.advance(1))
                         return Incomplete;
 
-                char* end;
-                const char* raw = chunkSize.rawText();
-                auto sz         = std::strtol(raw, &end, 16);
+                const char* beg = chunkSize.rawText();
+                while (*beg == '+' || *beg == ' ')
+                {
+                    *beg++;
+                }
+                std::string_view view = beg;
+                const char* end = view.data() + view.length();
+                long sz;
```

Comment:
To assist with debugging in case something breaks, please default value initialize `sz`.